Pod/Classes/PINRemoteImageManager.m

```diff
@@ -281,6 +283,21 @@ - (void)unlock
     [_lock unlock];
 }
 
+- (void)setValue:(nullable NSString *)value forHTTPHeaderField:(nullable NSString *)header {
+    __weak typeof(self) weakSelf = self;
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        typeof(self) strongSelf = weakSelf;
+        [strongSelf lock];
+            if (value) {
+                strongSelf.httpHeaderFields[header] = value;
+            }
+            else {
+                [strongSelf.httpHeaderFields removeObjectForKey:header];
```

Comment:
Do we need the else here? I was under the impression NSMutableDictionary's behavior was to remove the object if you set it to nil using the object subscripting stuff?